src/Flash.tsx

```diff
@@ -122,7 +122,9 @@ export const Flash = ({
 
   return (
     <div className={cls} style={style}>
-      <span className={`${stylePrefix}__value`}>{valueFormatter(value)}</span>
+      <span className={`${stylePrefix}__value`} data-testid="value">
```

Comment:
Are we sure we want `data-testid` here? Is there an accessible role for this element that makes sense?

Reply:
We can directly query for the value and test it against null.  I do personally like data-testid better.  There is also a babel plugin that we can use to remove these data-testid for production.  Let me know.

Reply:
I'm just thinking that maybe there is a more accessible way to write this HTML, and having to use `data-testid` is symptomatic of our code _not_ being accessible.

I can't think of a good way to do this off the top of my head... maybe there is a good accessible role that we can use for this element?

If we do have to use `data-id`, I think it would be good to explore that babel plugin... I wouldn't want consumers of this package to struggle with having such a generic name in _their_ UI tests.